src/attackmate/attackmate.py

```diff
@@ -60,6 +60,7 @@ def initialize_executors(self):
         self.executors = {
             'shell': executors.ShellExecutor(**init_args),
             'ssh': executors.SSHExecutor(**init_args),
+            'sftp': executors.SSHExecutor(**init_args),
```

Comment:
this will break some features(like caching) since we expected to use the same instance of the SSHExecutor for sftp and ssh

Reply:
used same instance of the ssh executor instead